Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -438,7 +438,15 @@ - (void)interstitialAdDidFailToLoad:(VungleInterstitial * _Nonnull)interstitial
 
 - (void)interstitialAdDidPresent:(VungleInterstitial * _Nonnull)interstitial
 {
-    [self.interstitialAdDelegate didDisplayInterstitialAd];
+    NSString *creativeIdentifier = interstitial.creativeId;
+    if ( ALSdk.versionCode >= 6150000 && [creativeIdentifier al_isValidString] )
```

Comment:
Might we define `6150000` as a constant as it appears many times? Thanks a lot. :)